components/Board/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import { Icon } from '@/components'
+import Image from 'next/image'
+import styles from '@/styles/Board.module.css'
+
+const Card = (props) => {
+    const { image, info } = props
+    return (
+        <div className={styles.card_box}>
+            <div className={styles.image_box}>
+                <Image src={image.src} alt={image.alt} width={image.width} height={image.height} />
+            </div>
+            <div className={styles.center_box}>
+                <div className={styles.title_box}>
+                    <h5 className={styles.title_text}>{info.title}</h5>
+                </div>
+                <div>
```

Comment:
If you are not identifying a div, you maybe not need it

Reply:
![image](https://user-images.githubusercontent.com/57446278/223595707-a03379b7-5ef4-45aa-8f46-337b9eba5610.png)

That's odd. I have a "p" tag inside

Reply:
Removed